src/component/feature/FeatureView2/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/FeatureOverviewEnvironmentStrategies/FeatureOverviewEnvironmentStrategies.tsx

```diff
@@ -0,0 +1,38 @@
+import { IFeatureStrategy } from '../../../../../../../interfaces/strategy';
+import FeatureOverviewEnvironmentStrategy from './FeatureOverviewEnvironmentStrategy/FeatureOverviewEnvironmentStrategy';
+
+interface FeatureOverviewEnvironmentStrategiesProps {
+    strategies: IFeatureStrategy[];
+    environmentName: string;
+}
+
+const FeatureOverviewEnvironmentStrategies = ({
+    strategies,
+    environmentName,
+}: FeatureOverviewEnvironmentStrategiesProps) => {
+    const renderStrategies = () => {
+        return strategies.map(strategy => {
+            return (
+                <FeatureOverviewEnvironmentStrategy
+                    key={strategy.id}
+                    strategy={strategy}
+                    environmentName={environmentName}
+                />
+            );
+        });
+    };
+
+    return (
+        <div
+            style={{
```

Comment:
Is there any reason to inline this style?

Reply:
Not really. I sometimes inline if there is not excessive need for styling to avoid creating a stylefile. Like here where we only have a containing div that needs three properties. It's a micro optimization and reduction of cognitive load.